core/src/main/java/nl/nn/adapterframework/senders/Samba2Sender.java

```diff
@@ -65,4 +68,9 @@ public void setRealm(String realm) {
 		getFileSystem().setRealm(realm);
 	}
 
```

Comment:
You replaced the IbisDoc of setPort() with IbisDocRef. Thanks for that.
Could you do the same for the other setters of Samba2Sender?
Could you do the same for Samba2Pipe too?